src/main/scala/org/apache/spark/deploy/armada/Config.scala

```diff
@@ -120,4 +133,91 @@ private[spark] object Config {
       .stringConf
       .checkValue(serviceNamePrefixValidator, "Service name prefix must adhere to rfc 1035 label names.")
       .createWithDefaultString("armada-spark-driver-")
+
+  private val invalidLabelListErrorMessage =
+    "Must be a comma-separated list of valid Kubernetes labels (each as key=value)"
+
+  val ARMADA_SPARK_GLOBAL_LABELS: ConfigEntry[String] =
+    ConfigBuilder("spark.armada.global.labels")
```

Comment:
I'd prefer `spark.armada.labels` or `spark.armada.pod.labels.

Reply:
My idea was to have an option to define global labels (which get applied to both driver and executors), driver labels and executor labels.

Are you refering only for this global case?

I like the `spark.armada.labels` approach, I'll do that in a follow-up.

Reply:
This is just a suggestion to rename the global config key as `globa` does not add much meaning to this key.